infrastructure/dev/README.md

```diff
@@ -0,0 +1,10 @@
+# Development infrastructure
+
+This directory contains the scaffolding to build and run a local Dockerized
+version of the Jitsi server chain. To build and run this Dockerized setup, a
+convenience script is provided at [`bin/jitsi-local-run`](../bin/jitsi-local-run).
+
+The "entry point" for this setup is the
+[`jitsi-meet.Dorckerfile`](jitsi-meet.Dockerfile), which specifies how to
+build a Docker image for a local Jitsi setup. This Dockerfile depends, in
+turn, on the shell scripts in this directory.
```

Comment:
Not blocking: I like that we have sprouted a folder for encapsulating how to run jitsi meet on a local machine; but I wonder if the naming is slightly off. It's not so much `dev` as it is `jitsi-meet-local`.